Trimble.Modus.Components/Controls/TMCheckBox/TMCheckBox.cs

```diff
@@ -0,0 +1,188 @@
+﻿using Microsoft.Maui.Controls;
+using System.Drawing;
+using Trimble.Modus.Components.Enums;
+
+namespace Trimble.Modus.Components
+{
+    public class TMCheckBox : ContentView
+    {
```

Comment:
Please refer TMInput and add regions and comments

Reply:
Add Regions and comments